lib/glific/flows/webhook.ex

```diff
@@ -181,15 +181,40 @@ defmodule Glific.Flows.Webhook do
     nil
   end
 
+  # THis function will create a dynamic headers
+  @spec create_headers(Action.t(), FlowContext.t()) :: map()
+  defp create_headers(action, context) do
+    default_payload = %{
```

Comment:
Let's not duplicate the code and create a common function to get these variables for parser.